roles/openshift_node_upgrade/tasks/rpm_upgrade.yml

```diff
@@ -12,3 +12,8 @@
 - name: Ensure python-yaml present for config upgrade
   package: name=PyYAML state=present
   when: not openshift.common.is_atomic | bool
+
+- name: Install Node service file
+  template:
+    dest: "/etc/systemd/system/{{ openshift.common.service_type }}-node.service"
+    src: "{{ openshift.common.service_type }}-node.service.j2"
```

Comment:
Probably better to put this in roles/openshift_node_upgrade/tasks/systemd_units.yml

I see other diffs between roles/openshift_node/tasks/systemd_units.yml and roles/openshift_node_upgrade/tasks/systemd_units.yml and I wonder if those should be pulled over too? @ingvagabund thoughts?